README.md

```diff
@@ -77,24 +119,22 @@ Configurations for tests and linting can be found in the `iati/` directory.
 
 ```
 # Run tests from the project root
-pytest iati/
+docker-compose run web pytest
```

Comment:
Something I learned recently. If you already have a daemon of the compose up (e.g. `docker-compose up -d`) then it's better to use `exec` instead of `run` (e.g. `docker-compose exec web pytest`). It turns out that `run` spins up a whole new set of containers each time you use it, while `exec` looks for an already running network to execute your command on.